package.json

```diff
@@ -25,6 +26,9 @@
     "postinstall-postinstall": "^2.1.0"
   },
   "resolutions": {
+    "@babel/helper-create-class-features-plugin": "^7.12.13",
+    "@babel/helper-replace-supers": "^7.12.13",
+    "@babel/plugin-transform-classes": "^7.12.13",
```

Comment:
Bovenstaande resolutions zijn nodig ivm https://github.com/whitespace-se/storybook-addon-html/issues/46: `@whitespace-se/storybook-addon-html` moet deze packages updaten, tot die tijd is dit een prima oplossing.